src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 0;
+
+  if (days < 3) {
+    rentalCost = days * 40;
+  } else {
+    rentalCost = days * 40 - 20;
+  }
```

Comment:
try to avoid this `if else`
you could calculate base cost and then minus discount (you will need 2 if's)